swarm/scheduler/filter.md

```diff
@@ -72,7 +73,7 @@ practical applications:
 
 * Schedule based on specific host properties, for example,`storage=ssd` schedules
   containers on specific hardware.
-* Force containers to run in a given location, for example region=us-east`.
+* Force containers to run on nodes in a given location, for example `region=us-east`.
```

Comment:
If I understand it right, then `us-east` is only a label, and does not in and of itself give you a location, unless you have labeled your nodes that way. That's what I was trying to convey when I asked you to change it to 'label'. I don't want people to think that it's magic. Does that make sense or am I missing something?